lib/phoenix/token.ex

```diff
@@ -113,7 +113,44 @@ defmodule Phoenix.Token do
   end
 
   @doc """
-  Decodes the original data from the token and verifies its integrity.
```

Comment:
The first line of the documentation must always be a short summary. So we should keep the line above or right a new one. :)